tests/test_images.py

```diff
@@ -6,12 +8,26 @@
 
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
-    remote = ("https://raw.githubusercontent.com/"
-              "komljen/dockerfile-examples/master/redis/Dockerfile")
+    remote = ("https://gist.githubusercontent.com/barrachri/"
+              "16a735971379352635a92d65036145f1/raw"
+              "/2b814ad3bf24af94a5623df03678816e37f2b6b7/Dockerfile")
     name = "image:1.0"
     await docker.images.build(tag=name, remote=remote)
-    image = await docker.images.get(name)
-    assert image
+    # build the image requires time
+    # we hold on for max 10 secs
+    # until we are sure the image
+    # is available locally using `get`
+    try:
+        with aiohttp.Timeout(10):
+            while True:
+                try:
```

Comment:
Not big problem, fell free to ignore, but using `contextlib.suppress` decorator  https://docs.python.org/3/library/contextlib.html#contextlib.suppress  makes `try/except` look nicer